packages/ui-extensions-react/src/surfaces/customer-account/hooks/app-metafields.ts

```diff
@@ -1,5 +1,5 @@
 import type {
-  RenderOrderStatusExtensionTarget,
```

Comment:
I can't remember why we had to prefix the order status extension targets. Should we keep this so that we don't need to cause another breaking type change in the future ?